src/UiPath.Workflow/Activities/ScriptingJitCompiler.cs

```diff
@@ -86,8 +86,25 @@ public override LambdaExpression CompileExpression(ExpressionToCompile expressio
         }
 
         return (LambdaExpression)results.ResultType.GetMethod("CreateExpression")!.Invoke(null, null);
+
+        TextExpressionCompilerResults TryBuild()
+        {
+            try
+            {
+                return ScriptingAotCompiler.BuildAssembly(finalCompilation, compilation.ScriptClass.Name, collectibleAlc);
+            }
+            catch
+            {
+                var errorResults = ScriptingAotCompiler.GetCompilerResults(compilation);
```

Comment:
Please do not use the compilation variable for getting diagnostics, as its purpose differs from the actual script (finalCompilation). You could throw a rather generic exception or so.